vulnerabilities/api.py

```diff
@@ -175,9 +176,9 @@ def filter_purl(self, queryset, name, value):
 
 
 class PackageViewSet(viewsets.ReadOnlyModelViewSet):
-    queryset = Package.objects.all()
+    queryset = Package.objects.all().order_by("id")
     serializer_class = PackageSerializer
-    paginate_by = 50
+    pagination.PageNumberPagination.page_size = 10
```

Comment:
This seems quite wrong. It looks like you are overriding a value for a completely different class reference.